uber/templates/groups/form.html

```diff
@@ -25,147 +25,194 @@
-</tr>
+    </div>
+</div>
+
+<div class="form-group">
+    <label class="col-sm-2 control-label">Can Add Badges</label>
+    <div class="col-sm-6">
+        {% checkbox group.can_add %} Allow this group to add badges
+    </div>
+</div>
+
 {% if new_dealer or group.is_dealer %}
     {% if group.leader %}
-        <tr>
-            <td> <b>Phone Number</b> </td>
-            <td> {{ group.leader.cellphone }} </td>
-        </tr>
+        <div class="form-group">
+            <label class="col-sm-2 control-label">Phone Number</label>
+            <div class="col-sm-6">
+                {{ group.leader.cellphone }}
+            </div>
+        </div>
     {% endif %}
-    <tr>
-        <td valign="top"> <b>Status</b> </td>
-        <td id="status">
+
+    <div class="form-group">
+        <label class="col-sm-2 control-label">Status</label>
+        <div id="status" class="col-sm-6">
             {% if new_dealer %}
                 <select name="status">
-                    <option value="{{ APPROVED }}">Approved</option>
-                    <option value="{{ UNAPPROVED }}">Unapproved</option>
-                    <option value="{{ WAITLISTED }}">Waitlisted</option>
+                    <option value="{{ c.APPROVED }}">Approved</option>
+                    <option value="{{ c.UNAPPROVED }}">Unapproved</option>
+                    <option value="{{ c.WAITLISTED }}">Waitlisted</option>
                 </select>
             {% else %}
                 {{ group.status_label }} <br/>
-                {% if group.status != APPROVED %}
-                    <input type="checkbox" name="status" value="{{ APPROVED }}" /> Approved
+                {% if group.status != c.APPROVED %}
+                    <input type="checkbox" name="status" value="{{ c.APPROVED }}" /> Approved
                     <a href="#" onClick="$('#setstatus').toggle(); return false;">Waitlist or Reject</a> <br/>
                     <div id="setstatus" style="display:none">
                         Enter an email message to be sent along with the notification: <br/>
-                        (The email subject will be "Your {{ EVENT_NAME }} Dealer registration has been [waitlisted | declined]")
+                        (The email subject will be "Your {{ c.EVENT_NAME }} Dealer registration has been [waitlisted | declined]")
                         <textarea name="email", rows="5" cols="50"></textarea> <br/>
                         <button onClick="unapprove('waitlisted'); return false;">Waitlist</button>
                         <button onClick="unapprove('declined'); return false;">Reject</button>
                     </div>
                 {% endif %}
             {% endif %}
-        </td>
-    </tr>
-    <tr>
-        <td valign="top"> <b>What do they sell?</b> </td>
-        <td> <textarea rows="3" cols="30" name="wares">{{ group.wares }}</textarea>
-    </tr>
-    <tr>
-        <td> <b>Tax ID</b> </td>
```

Comment:
Is there any way for plugins to insert fields dynamically on the admin side, like for these fields here?